src/JsonConfig.cpp

```diff
@@ -37,46 +38,30 @@ JsonConfig::JsonConfig(const QString &path) :
 
 bool JsonConfig::load()
 {
-    QFile file(m_path);
-    if (!file.open(QFile::ReadOnly)) {
-        return false;
-    }
-
-    QJsonDocument doc = QJsonDocument::fromJson(file.readAll());
-    QJsonValue schemaVersionsRaw = doc.object().value("schemaVersions");
-    if (!schemaVersionsRaw.isArray()) {
-        qWarning() << "Invalid schemaVersions";
+    QDir dir(m_path);
+    dir.setFilter(QDir::AllEntries | QDir::NoDotAndDotDot);
+    int total_files = dir.count();
+    if (!dir.exists() || total_files < 1)
+    {
         return false;
     }
-
-    QJsonArray schemaVersions = schemaVersionsRaw.toArray();
-    QJsonArray::iterator it = schemaVersions.begin();
-    for (; it != schemaVersions.end(); ++it) {
-        QJsonValue val = *it;
-
-        if (!val.isObject()) {
-            continue;
-        }
-
-        QJsonObject schemaVersion = val.toObject();
-        QJsonValue version = schemaVersion.value("version");
-        if (!version.isDouble()) {
-            continue;
+    QStringList migrations = dir.entryList(QStringList() << "*.sql" << "*.SQL", QDir::Files);
+    foreach(QString filename, migrations)
+    {
+        QString filePath = dir.absolutePath().append("/").append(filename);
```

Comment:
Use `filePath`: https://doc.qt.io/qt-5/qdir.html#filePath

Reply:
Changed to use filePath.